### PR TITLE
Fixed using bin paths in local recipe

### PR DESCRIPTION
**Pull request by oanhnn**
| Q             | A
| ------------- | ---
| Bug fix?      | Yes
| New feature?  | No
| BC breaks?    | No
| Deprecations? | No
| Fixed tickets | #46

### CODE REVIEW

File: docs/local.md [hunk 1]
```diff
@@ -26,6 +26,14 @@ env('local_deploy_path', '/tmp/deployer');
 - **local_release_path** - Accepts *string* specifying from where should deploys be done. This is functionally identical with `release_path` of `common.php` recipe. If You use full local release scenario, You do **not** wish to set this, as it's relative to `local_deploy_path`.
 - **local_git_cache** - Accepts *boolean*. By default it checks if Your git version supports required options (minimum git version is 2.3). If You do not use release workflow, You would not see benefits to this option. Set this to false to use shallow clones, that slims down cloned repository size.
 
+#### Locally bin paths
+
+> Auto detectors have non-UNIX OS problems, so we are highly recommended using your bin paths instead of it.
```
Comment (mbrodala) [inline docs/local.md]: Please change to "Autodetection has issues with non-UNIX OS, thus you are highly recommended to use your own paths instead." Also in the comment below.

File: docs/local.md [hunk 2]
```diff
@@ -26,6 +26,14 @@ env('local_deploy_path', '/tmp/deployer');
...
+
+- **local_bin/php** - Accepts *string*. By default it checks if your locally php bin path, but it only run on UNIX OS.
```
Comment (mbrodala) [inline docs/local.md]: Please change to "... it checks for your local PHP bin path, which only works on UNIX OS"

File: docs/local.md [hunk 3]
```diff
@@ -26,6 +26,14 @@ env('local_deploy_path', '/tmp/deployer');
...
+- **local_bin/git** - Accepts *string*. By default it checks if your locally git bin path, but it only run on UNIX OS.
```
Comment (mbrodala) [inline docs/local.md]: Please change to "... it checks for your local Git bin path, which only works on UNIX OS"

File: docs/local.md [hunk 4]
```diff
@@ -26,6 +26,14 @@ env('local_deploy_path', '/tmp/deployer');
...
+- **local_bin/composer** -Accepts *string*. By default it checks if your locally composer bin path, but it only run on UNIX OS.
```
Comment (mbrodala) [inline docs/local.md]: Please change to "... it checks for your local Composer bin path, which only works on UNIX OS"

**Comment by mathieu-gilloots** [comment]
@oanhnn Could you do the some fixes to merge this PR, I need it on my projects :)

**Comment by oanhnn** [comment]
@mathieu-gilloots Ok. I will fix it and merge.

**Comment by mathieu-gilloots** [comment]
Thank you so much it will be useful :+1:

**Pull request merged**